src/community/english.ts

```diff
@@ -19,17 +18,17 @@ const PREFIXES = [
 ];
 
 const PREFIXES_2 = [
-  "", "milli-", "micri-", "nani-", "pici-", "femti-", "atti-", "zepti-", "yocti-", "xoni-",
-  "veci-", "meci-", "dueci-", "treci-", "tetreci-", "penteci-", "hexeci-", "hepteci-", "octeci-", "enneci-",
-  "icosi-", "mecosi-", "duecosi-", "trecosi-", "tetrecosi-", "pentecosi-", "hexecosi-", "heptecosi-", "octecosi-", "ennecosi-",
-  "triaconti-", "metriaconti-", "duetriaconti-", "tretriaconti-", "tetretriaconti-", "pentetriaconti-", "hexetriaconti-", "heptetriaconti-", "octtriaconti-", "ennetriaconti-",
-  "tetraconti-", "metetraconti-", "duetetraconti-", "tretetraconti-", "tetretetraconti-", "pentetetraconti-", "hexetetraconti-", "heptetetraconti-", "octetetraconti-", "ennetetraconti-",
-  "pentaconti-", "mepentaconti-", "duepentaconti-", "trepentaconti-", "tetrepentaconti-", "pentepentaconti-", "hexepentaconti-", "heptepentaconti-", "octepentaconti-", "ennepentaconti-",
-  "hexaconti-", "mehexaconti-", "duehexaconti-", "trehexaconti-", "tetrehexaconti-", "pentehexaconti-", "hexehexaconti-", "heptehexaconti-", "octehexaconti-", "ennehexaconti-",
-  "heptaconti-", "meheptaconti-", "dueheptaconti-", "treheptaconti-", "tetreheptaconti-", "penteheptaconti-", "hexeheptaconti-", "hepteheptaconti-", "octeheptaconti-", "enneheptaconti-",
-  "octaconti-", "meoctaconti-", "dueoctaconti-", "treoctaconti-", "tetreoctaconti-", "penteoctaconti-", "hexeoctaconti-", "hepteoctaconti-", "octeoctaconti-", "enneoctaconti-",
-  "ennaconti-", "mecennaconti-", "duecennaconti-", "trecennaconti-", "tetrecennaconti-", "pentecennaconti-", "hexecennaconti-", "heptecennaconti-", "octecennaconti-", "ennecennaconti-",
-  "hecti-", "mehecti-", "duhecti-",
+  "", "milli-", "micro-", "nano-", "pico-", "femto-", "atto-", "zepto-", "yocto-", "xono-",
+  "veco-", "meco-", "dueco-", "treco-", "tetreco-", "penteco-", "hexeco-", "hepteco-", "octeco-", "enneco-",
+  "icoso-", "meicoso-", "dueicoso-", "trioicoso-", "tetreicoso-", "penteicoso-", "hexeicoso-", "hepteicoso-", "octeicoso-", "enneicoso-",
```

Comment:
Codacy found an issue: [Exceeds maximum line length of 120](https://app.codacy.com/gh/antimatter-dimensions/notations/pullRequest?prid=6279527)